### PR TITLE
Release 1.0.2

### PR DESCRIPTION
**Pull request by aCampello**
- [x] Change `wellcomeml/__version__.py`
- [x] Add changelog
- [x] `make dist`
- [x] Verify new package was generated correctly on the [pip registry](https://pypi.org/project/wellcomeml)
 and GitHub releases

### CODE REVIEW

**Review by nsorros — APPROVED** [review]
LGTM 👍

File: create_release.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+git fetch && git rebase origin/master
```
Comment (nsorros) [inline create_release.sh:2]: maybe you want to ensure you are in master. an alternative is to change that to `git fetch && git reset --hard origin/master` in which case it won't matter.

Comment (aCampello) [reply create_release.sh:2]: I don't it! I'm pushing for `release-1.0.2`, I think if I reset-hard I'll lose all the commits from this branch

Comment (aCampello) [reply create_release.sh:2]: Ah, I see what you mean. I could alternatively checkout to master? 

```git fetch && git checkout/master && git reset --hard origin/master```?

Comment (nsorros) [reply create_release.sh:2]: I thought you were doing it from master, which makes no sense. Leave as is.

**Comment by aCampello** [comment]
```pip install wellcomeml --no-cache-dir``` outputs the new version. Just for a sanity check, the HDbscan bug is gone, tested using

``` python -c "from wellcomeml.ml import CNNClassifier"```

**Pull request merged**